src/sentry/tasks/backfill_seer_grouping_records.py

```diff
@@ -421,18 +421,23 @@ def lookup_group_data_stacktrace_bulk(
                 )
                 event = Event(event_id=event_id, project_id=project_id, group_id=group_id)
                 event.data = data
-                if event and event.data and event.data.get("exception") and hashes.get(group_id):
+                if event and event.data and event.data.get("exception"):
                     grouping_info = get_grouping_info(None, project=project, event=event)
                     stacktrace_string = get_stacktrace_string(grouping_info)
                     if stacktrace_string == "":
                         invalid_event_ids.add(event_id)
                         continue
+                    primary_hash = event.get_primary_hash()
+                    if not primary_hash:
+                        invalid_event_ids.add(event_id)
+                        continue
+
                     group_data.append(
                         CreateGroupingRecordData(
                             group_id=group_id,
                             project_id=project_id,
                             message=messages[group_id],
-                            hash=hashes[group_id],
+                            hash=event.get_primary_hash(),
```

Comment:
can we just use `primary_hash` instead of re-calling `event.get_primary_hash()`?

Reply:
fixed!